makeflow/src/makeflow.c

```diff
@@ -1405,7 +1408,6 @@ int main(int argc, char *argv[])
 		{"gc", required_argument, 0, 'g'},
 		{"gc-size", required_argument, 0, LONG_OPT_GC_SIZE},
 		{"gc-count", required_argument, 0, 'G'},
-		{"wait-for-files-upto", required_argument, 0, LONG_OPT_FILE_CREATION_PATIENCE_WAIT_TIME},
```

Comment:
Yes. For some reason probably due to an old code change this entry in is the struct twice. The first is on line 1404/1407.